<launch>
     <node pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" name="depthimage_to_laserscan" >
          <param name="scan_height" value="1" />   <!--default value: 1 (pixel) -->
          <param name="scan_time" value="0.033" />   <!--default value: 0.033 (seconds)-->
          <param name="range_min" value="0.1" />   <!--default value: 0.45 (meters) -->
          <param name="range_max" value="10.0" />   <!--default value: 10.0 (meters) -->
          <param name="output_frame_id" value="camera_depth_frame" />   <!--default value: camera_depth_frame (topic string) -->

          <remap from="image" to="camera/depth/image_rect_raw" />
          <remap from="camera_info" to="camera/depth/camera_info" />
          <!-- <remap from="scan" to="camera_scan"/> -->
     </node>

     <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen" clear_params="true">
          <param name="max_iterations" value="10" />
          <param name="fixed_frame" value="odom" />
          <param name="publish_tf" value="true" />
          <param name="base_frame" value="base_link" />
          <param name="publish_pose" value="false" />
          <param name="publish_pose_stamped" value="true" />
          <param name="use_imu" value="true" />
          <param name="use_odom" value="false" />
          <param name="use_cloud_input" value="false" />
          <remap from="imu/data" to="imu/data/filtered"/>
          <remap from="cloud" to="camera/depth_registered/points"/>
     </node>

</launch>
